display.go

```diff
@@ -83,6 +94,109 @@ func updateStaticContents() {
 	updateLabelIfChanged("ui_Status_Content_Device_Id_Content_Label", GetDeviceID())
 }
 
+// setDisplayBrightness sets /sys/class/backlight/backlight/brightness to alter
+// the backlight brightness of the JetKVM hardware's display.
+func setDisplayBrightness(brightness int) error {
+	// NOTE: The actual maximum value for this is 255, but out-of-the-box, the value is set to 64.
+	// The maximum set here is set to 100 to reduce the risk of drawing too much power (and besides, 255 is very bright!).
+	if brightness > 100 || brightness < 0 {
+		return errors.New("brightness value out of bounds, must be between 0 and 100")
+	}
+
+	// Check the display backlight class is available
+	if _, err := os.Stat(BACKLIGHT_CONTROL_CLASS); errors.Is(err, os.ErrNotExist) {
+		return errors.New("brightness value cannot be set, possibly not running on JetKVM hardware")
+	}
+
+	// Set the value
+	bs := []byte(strconv.Itoa(brightness))
+	err := os.WriteFile(BACKLIGHT_CONTROL_CLASS, bs, 0644)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("display: set brightness to %v", brightness)
```

Comment:
the convention seems to be using `log.Printf()` and not `fmt.Printf()` in this code base.

Reply:
There's about three different logging conventions in the codebase:
![image](https://github.com/user-attachments/assets/e97c450f-5c32-4722-81d0-5b9a8f9d9bc2)

I was mainly going off the convention in `display.go` when I decided to use fmt, I just forgot the \n on that line.

Reply:
Hopefully a standard logging method is chosen, will refactor as part of that, just following the scheme used elsewhere in Display.go for now.